### ISSUE TITLE
Ability to export string values

### ISSUE THREAD

**Issue by antonblr** [oliver006/redis_exporter#523]
**Describe the problem**

More of a feature request. 

Right now there is no way to export string values from Redis - neither with `check-keys` flag nor with custom `lua` script. Every string value will be lost during `strconv.ParseFloat(stringVal, 64)` conversion.

Would be nice to have something similar to [kubernetes/kube-state-metrics/internal/store/pod.go#L1128](https://github.com/kubernetes/kube-state-metrics/blob/c36d9c6adca9b61b33398f6bcf06122564da2f95/internal/store/pod.go#L1128), where, for example, it exports K8S pods labels name/value pairs as the Prometheus metric labels and sets the metric itself to 1.

It will look like that in Prometheus (query result for `kube_pod_labels` metric): 

```
kube_pod_labels{container="kube-rbac-proxy-main", instance="10.10.74.97:8443", job="kube-state-metrics",
label_app="analytics-worker", label_kind="worker", label_repo="analytics", namespace="default", 
pod="analytics-worker-67c2dbtr"} 1
```

**What version of redis_exporter are you running?**
`1.23.1`

**Comment by oliver006** [comment]
Thanks for raising this issue @antonblr - this sounds useful.
I have no immediate plans for implementing this but would accept a PR.

**Comment by oliver006** [comment]
Added this as part of #607 - released as [1.35.0](https://github.com/oliver006/redis_exporter/releases/tag/v1.35.0)
